### PR TITLE
Use mariadb repo to install mariadb server

### PR DESCRIPTION
**Pull request by maelk**
upgrades mariadb server to 10.4.8 and fixes a bug
when running in Minikube

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM docker.io/centos:centos7
 
 RUN yum install -y python-requests && \
+    curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash && \
```
Comment (stbenjam) [inline Dockerfile]: Do you have a link to the bug? El8 has 10.3.11, want to make sure whatever bug we're hitting is fixed in the EL8 version so we're ready when the time comes.  There's no EL8 packages provided by this script:


```
  supported="The MariaDB Repository supports these Linux OSs, on x86-64 only:
      * RHEL/CentOS 6 & 7
      * Ubuntu 14.04 LTS (trusty), 16.04 LTS (xenial), & 18.04 LTS (bionic)
      * Debian 8 (jessie), 9 (stretch), & 10 (buster)
      * SLES 12 & 15"
```

Comment (maelk) [reply Dockerfile]: Unfortunately I don't :/ we did not find the actual bug

Comment (maelk) [reply Dockerfile]: you can find out very easily if it has the bug or not by running the docker container in Minikube 1.5.0. if it starts, the bug was fixed

**Comment by hardys** [comment]
I wonder if this is the same issue as https://github.com/kubernetes/minikube/issues/5751 - there are workarounds there, @maelk can you perhaps confirm if those fix this issue without upgrading mariadb?

**Comment by maelk** [comment]
The workaround fixes the issue. I'm going to open a PR towards metal3-dev-env to take the work-around in use temporarily, until they release Minikube 1.5.1

**Comment by maelk** [comment]
Fixed in Minikube

**Pull request closed without merging**